### PR TITLE
brings back mass grinding

### PR DESCRIPTION
**Pull request by Slorbin**
you broke it so i made a replacement. new grindstome item

### CODE REVIEW

File: code/rt.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #ifndef TESTING
-//    #define FASTLOAD
+    #define FASTLOAD
```
Comment (Gyrandola) [inline code/rt.dm:2]: oi oi oi

Comment (DeerJesus) [reply code/rt.dm:2]: im a duimbass

**Comment by Gyrandola** [comment]
This PR needs a better description. See the [contribution guidelines](https://github.com/Rotwood-Vale/Ratwood-Keep/blob/main/CONTRIBUTING.md).
Is this meant to replace all existing grindstones? If so you should take care of the map.

**Comment by DeerJesus** [comment]
> This PR needs a better description. See the [contribution guidelines](https://github.com/Rotwood-Vale/Ratwood-Keep/blob/main/CONTRIBUTING.md). Is this meant to replace all existing grindstones? If so you should take care of the map.

dog we have contribution guidelines now? lol

no it just adds it as a crafting recipe as a bandaid fix until we find out what we're actually fucking doing with reagents

**Comment by Nakalon** [comment]
> > This PR needs a better description. See the [contribution guidelines](https://github.com/Rotwood-Vale/Ratwood-Keep/blob/main/CONTRIBUTING.md). Is this meant to replace all existing grindstones? If so you should take care of the map.
> 
> dog we have contribution guidelines now? lol
> 
> no it just adds it as a crafting recipe as a bandaid fix until we find out what we're actually fucking doing with reagents

yes. read them please, they include a requirement of test footage. Makes our job much easier.

**Comment by DeerJesus** [comment]
> > > This PR needs a better description. See the [contribution guidelines](https://github.com/Rotwood-Vale/Ratwood-Keep/blob/main/CONTRIBUTING.md). Is this meant to replace all existing grindstones? If so you should take care of the map.
> 
> > 
> 
> > dog we have contribution guidelines now? lol
> 
> > 
> 
> > no it just adds it as a crafting recipe as a bandaid fix until we find out what we're actually fucking doing with reagents
> 
> 
> 
> yes. read them please, they include a requirement of test footage. Makes our job much easier.

It's just the pre-removal millstone to a T. Can't take a photo of it as I'm far away from my pc. But just trust me guyz(TM) it works

**Comment by Gyrandola** [comment]
It is still not clear to me why you are keeping both the millstone and the grindstone. Write a clear description about what you are adding and why.

**Comment by DeerJesus** [comment]
Millstone is for making abstract essence things. Grindstone is for mass
reagent extraction.

**Comment by DeerJesus** [comment]
> It is still not clear to me why you are keeping both the millstone and the grindstone. Write a clear description about what you are adding and why, or i'll close this.

is the above comment not clear enough

Grindstone is pre-NeuFood millstone function: mass mortar and pestle items.

Millstone is post-neufood what is on the server now: some weird shit for grinding certain things into the alchemy system

**Review by Gyrandola — APPROVED** [review]

**Pull request merged**